src/test/java/pl/jablonskanycz/bakery/database/services/AddressServiceTest.java

```diff
@@ -0,0 +1,100 @@
+package pl.jablonskanycz.bakery.database.services;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import pl.jablonskanycz.bakery.database.domain.AddressEntity;
+import pl.jablonskanycz.bakery.database.repositories.AddressRepository;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+@ActiveProfiles("test")
+class AddressServiceTest {
+    @Autowired
+    private AddressService addressService;
+
+    @MockBean
+    private AddressRepository addressRepository;
+
+    @Test
+    public void shouldReturnAllAddresses() {
+        //given
+        AddressEntity address1 = AddressEntity.builder().addressId(1L).latitude(51.20).longitude(15.80).build();
```

Comment:
do we have any lines that repeat within this file that we could extract to a method:)?